packaging/nnstreamer.spec

```diff
@@ -987,6 +993,7 @@ cp -r result %{buildroot}%{_datadir}/nnstreamer/unittest/
 
 %files misc
 %{gstlibdir}/libgstjoin.so
+%{gstlibdir}/libgstmqtt.so
```

Comment:
this also requires condition `%if 0%{?mqtt_support}`

Reply:
@jaeyun-jung Fixed. Thanks!